pkg/tsdb/loki/streaming.go

```diff
@@ -84,15 +82,9 @@ func (s *Service) RunStream(ctx context.Context, req *backend.RunStreamRequest,
 	params := url.Values{}
 	params.Add("query", query.Expr)
 
-	lokiDataframeApi := s.features.IsEnabled(featuremgmt.FlagLokiDataframeApi)
```

Comment:
should this just check the lokiLive flag instead?  so the stream endpoint still works?

Reply:
hi @ryantxu , this PR is basically:
"replace `lokiDataFrameApi` feature-flag with `true`, and remove the feature-flag".

currently it's not possible to use the direct format (i assume you mean the loki-emits-a-dataframe mode) anywhere except the `mode=streaming` option, and this PR is not regressing on that aspect... but maybe i misunderstood something?